BraintreeCore/src/main/java/com/braintreepayments/api/BraintreeClient.java

```diff
@@ -331,4 +360,21 @@ Context getApplicationContext() {
     public void invalidateClientToken() {
         authorizationLoader.invalidateClientToken();
     }
+
+    boolean launchesBrowserSwitchAsNewTask() {
+        return launchesBrowserSwitchAsNewTask;
+    }
+
+    /**
+     * Set this property to true to enable default deep link handling for browser switched flows.
+     * For web flows, this means launching the browser in a task separate from the calling activity.
+     *
+     * NOTE: When default deep linking is enabled, all custom url schemes set in {@link BraintreeClient}
+     * constructors will be ignored.
+     *
+     * @param launchesBrowserSwitchAsNewTask set to true to enable the default deep link handler. This value is false by default.
+     */
```

Comment:
Nitpick - I think we can reword this to remove `default` references, given the new property name.